tests/test_prompt_parser_validation.py

```
@@ -0,0 +1,121 @@
+"""Validation tests to ensure prompts and parsers are compatible."""
+
+import re
+from mad_spark_alt.core.qadi_prompts import QADIPrompts
+from mad_spark_alt.core.simple_qadi_orchestrator import SimpleQADIOrchestrator
+
+
+class TestPromptParserValidation:
+    """Tests to validate that prompts and parsers are compatible."""
+
+    def test_deduction_prompt_format_matches_parser_expectations(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_deduction_prompt_format_matches_parser_expectations`

Add return type annotation: `None`

(ANN201)